virtio-devices/src/iommu.rs

```diff
@@ -466,7 +456,6 @@ impl Request {
                         .memory()
                         .read_obj(req_addr as GuestAddress)
                         .map_err(Error::GuestMemory)?;
-                    debug!("Map request {:?}", req);
```

Comment:
We should probably keep the existing debug logs in the iommu package.

Reply:
My mistake, I thought this was my debug log - will readd this